MovieApp/MovieApp/MoviesViewController.m

```diff
@@ -19,8 +26,17 @@ @implementation MoviesViewController
 - (void)viewDidLoad {
     
     [super viewDidLoad];
+    [self configureView];
+
+    downloader = [[MovieDBDownloader alloc] init];
+    [downloader configure];
```

Comment:
Configuration part should be moved to AppDelegate and initialized on app startup only once for all mappings.